arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2186,15 +2146,34 @@
       },
       "discriminator": "objectType"
     },
-    "WorkloadProtectableItem": {
-      "description": "Base class for backup item. Workload-specific backup items are derived from this class.",
-      "type": "object",
+    "WorkloadProtectableItemResource": {
```

Comment:
Ok, no worries, just wanted to be sure on that. 

I'll review it this morning.